.github/workflows/publish-docs.yaml

```diff
@@ -12,6 +12,10 @@ name: Publish Docs
 on:
   # Triggers the workflow on push request events for all branches
   push:
+    paths:
+      - 'doc/**'
+      - 'libs/**'
+      - 'src/libAtomVM/**'
 
```

Comment:
We probably want to also include:
- .github/workflows

Not sure if we also need CMakeLists.txt

Reply:
This needed both.